user-guide/Reference/DataMiner_Tools/Ping_Monitoring/Installing_Ping_Monitoring.md

```diff
@@ -0,0 +1,50 @@
+---
+uid: Installing_Ping_Monitoring
+---
+
+# Installing the Ping Monitoring tool
+
+## Prerequisites
+
+- DataMiner version 10.3.9 [CU2] or higher
+
+  > [!IMPORTANT]
+  > The DataMiner Ping Monitoring tool was tested to be compatible with DataMiner versions 10.3.9 [CU2] and higher. We do not recommend using an older version as this may lead to unexpected issues.
+
+- A DataMiner System [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud)
+
+- The [*Generic Ping* connector](https://catalog.dataminer.services/result/driver/530)
+
+  > [!TIP]
+  > We recommend using Generic Ping version 3.1.2.11 or higher. See [Deploying a DataMiner connector to your system](xref:Deploying_A_DataMiner_Connector_to_your_system).
+
+- At least one Generic Ping element within your DMS that is configured with one or multiple destinations
+
+  > [!TIP]
+  >
+  > - For more information about configuring destinations, consult the [Generic Ping connector help](https://catalog.dataminer.services/result/driver/530).
+  > - See also: [Working with elements in DataMiner](xref:Element_cards)
+
+- Make sure the following soft-launch options are enabled:
```

Comment:
Perhaps we should adjust this a bit to make it more future-proof. For instance:

```suggestion
- Depending on your DataMiner version, you may need to enable the following soft-launch options:
```

Reply:
The following soft-launch options are needed when using DataMiner version 10.3.9 [CU2]. Future versions of DataMiner may include the necessary features. You can check the release status of these options here https://docs.dataminer.services/user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.html